test/unit/require.spec.ts

```diff
@@ -33,4 +33,20 @@ export class RequireTests {
         }
     }
 
+    @TestCase("", "src.fake")
+    @TestCase("/** @noResolution */", "fake")
+    @Test("noResolution on ambient modules causes no path alterations")
+    public testRequireModule(comment: string, expectedPath: string): void {
+        const regex = /require\("(.*?)"\)/;
```

Comment:
Try to lay out your tests according to AAA (Arrange/Act/Assert), this regex is part of the last (assertion) step, so I'd move this to right before where you actually use it.